src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -300,8 +300,15 @@ void GcsPlacementGroupManager::OnPlacementGroupCreationSuccess(
   auto placement_group_id = placement_group->GetPlacementGroupID();
   RAY_CHECK_OK(gcs_table_storage_->PlacementGroupTable().Put(
       placement_group_id, placement_group->GetPlacementGroupTableData(),
-      [this, placement_group_id](Status status) {
+      [this, placement_group_id, placement_group](Status status) {
         RAY_CHECK_OK(status);
+        // Update the resource in gcs resource manager
+        for (auto &bundle : placement_group->GetBundles()) {
+          auto &resources = bundle->GetFormattedResources();
+          auto node_id = bundle->NodeId();
+          gcs_resource_manager_.UpdateResources(node_id, resources);
```

Comment:
Btw, would there be any possibility there are unsynced code path between GCS & raylet (e.g., raylet doesn't have resources, but GCS does).

Maybe we should also consolidate created/deleted resources from resource report as well?

Reply:
I think it's not changing the original protocol. If it's from resource reporter, it'll still come from that.